crates/rolldown/src/ast_scanner/mod.rs

```diff
@@ -52,12 +52,12 @@ pub struct AstScanner<'me> {
   symbols: &'me mut AstSymbols,
   current_stmt_info: StmtInfo,
   result: ScanResult,
-  esm_export_keyword: Option<Span>,
-  esm_import_keyword: Option<Span>,
+  esm_export_keyword_start: Option<u32>,
```

Comment:
Since all length of `keyword` and special `identifier` is fixed, just save their start offset is enough

Reply:
Could you storing span to keep consistency align with others?

I'm not sure this actually save anything, since `CjsExportStartOffset` also takes u64 bytes.

Reply:
What do you mean by u64 bytes?

Reply:
64 bytes?

Reply:
or size of `u64`?

Reply:
FYI, store span rather than with enum also allocates 4 more bytes, they don't cost the same memory.

Reply:
or size of u64